README.md

```diff
@@ -187,21 +192,59 @@ export default Route.extend(InfinityRoute, {
     <h2>{{product.description}}</h2>
   {{/each}}
 
-  {{infinity-loader infinityModel=model.products infinityLoad=(action "loadMoreProduct")}}
+  {{infinity-loader infinityModel=model.products}}
 <section>
 ```
 
-The ability to use closure actions will be available in the `1.0.0-beta` series.  Also, this method uses Controllers.  Despite what you may have heard, Controllers are a great primitive in Ember's ecosystem.  Their singleton nature is great for handling queryParams and actions propagated from somewhere in your component tree.
-
 
 ### Service Methods
 
-The infinity-loader service exposes 4 methods:
+The infinity service also exposes 5 methods to fetch & mutate your collection:
+
+1. model
+2. replace
+3. flush
+4. pushObjects
+5. unshiftObjects
```

Comment:
Q - how do these interplay with the `infinityCache`?

Reply:
Hm.  Right now it is just updating the underlying collection passed from the user context.  To keep that in sync, we would need to find it in the cachedCollection.  However, I'm not totally sure how to find `which` one atm.

Reply:
Sry this is pass by reference.  So they will be in sync!  Added a test to show this behavior as well.